_aggregations/pipeline/percentiles-bucket.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: default
+title: Percentiles bucket
+parent: Pipeline aggregations
+nav_order: 160
+redirect_from:
+  - /query-dsl/aggregations/pipeline/percentiles-bucket/
+---
+
+# Percentiles bucket aggregations
+
+The `percentiles_bucket` aggregation is a sibling aggregation that calculates the percentile placement of bucketed metrics.
+
+The `percentiles_bucket` does not interpolate to calculate percentiles. Each percentile value is the closest value that is less than or equal to that percentile.
```

Comment:
```suggestion
The `percentiles_bucket` aggregation does not interpolate to calculate percentiles. Each percentile value is the closest value that is less than or equal to that percentile.
```

Reply:
This sentence and the next one are related. Maybe combine them into one? 

The `percentiles_bucket` aggregation computes percentiles exactly, without approximation or interpolation. Each percentile is returned as the closest value less than or equal to the target percentile.

Reply:
```suggestion
The `percentiles_bucket` aggregation computes percentiles exactly, without approximation or interpolation. Each percentile is returned as the closest value less than or equal to the target percentile.
```